conf/app.routes

```diff
@@ -1,22 +1,29 @@
 POST  /org/:orgRef/awrs/send-data                                    controllers.OrgSubscriptionController.subscribe(orgRef: String)
 POST  /sa/:utr/awrs/send-data                                        controllers.SaSubscriptionController.subscribe(utr: String)
+POST  /awrs/send-data                                                controllers.GenericSubscriptionController.subscribe(ref: String ?= "")
 
 GET   /org/:orgRef/awrs/lookup/:awrsRefNo                            controllers.OrgSubscriptionController.lookupApplication(orgRef: String, awrsRefNo: String)
 GET   /sa/:utr/awrs/lookup/:awrsRefNo                                controllers.SaSubscriptionController.lookupApplication(utr: String, awrsRefNo: String)
+GET   /awrs/lookup/:awrsRefNo                                        controllers.GenericSubscriptionController.lookupApplication(ref: String ?= "", awrsRefNo: String)
 
 GET   /org/:orgRef/awrs/status/:awrsRefNo                            controllers.OrgSubscriptionController.checkStatus(orgRef: String, awrsRefNo: String)
 GET   /sa/:utr/awrs/status/:awrsRefNo                                controllers.SaSubscriptionController.checkStatus(utr: String, awrsRefNo: String)
+GET   /awrs/status/:awrsRefNo                                        controllers.GenericSubscriptionController.checkStatus(ref: String ?= "", awrsRefNo: String)
 
 PUT   /org/:orgRef/awrs/update/:awrsRefNo                            controllers.OrgSubscriptionController.updateSubscription(orgRef: String, awrsRefNo: String)
 PUT   /sa/:utr/awrs/update/:awrsRefNo                                controllers.SaSubscriptionController.updateSubscription(utr: String, awrsRefNo: String)
+PUT   /awrs/update/:awrsRefNo                                        controllers.GenericSubscriptionController.updateSubscription(utr: String ?= "", awrsRefNo: String)
```

Comment:
Should this not be utr and be ref: String ?

Reply:
It doesn't strictly matter I think, but I can change it for consistency